feature_server/server.py

```diff
@@ -205,7 +207,16 @@ def on_block_build_attempt(self, x, y, z):
         return self._can_build()
 
     def on_line_build_attempt(self, points):
-        return self._can_build()
+        if not self._can_build():
```

Comment:
oooh i see. So `on_line_build_attempt` must onnly ever be checked for false as well... weird